aesara/tensor/random/op.py

```diff
@@ -399,3 +399,43 @@ def grad(self, inputs, outputs):
 
     def R_op(self, inputs, eval_points):
         return [None for i in eval_points]
+
+
+class RandomMakerOp(Op):
+    def make_node(self, seed=None):
+        # np.inf is used as an representative of seed = None
+        if seed is None:
+            seed = np.inf
```

Comment:
`None` can be used in a graph (see `aesara.tensor.type_other.NoneConst`), and we should use it here.